selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -205,6 +205,18 @@ DevicePanel::DevicePanel(SettingsWindow *parent) : ListWidget(parent) {
 
   // offroad-only buttons
 
+  auto tskBtn = new ButtonControl(tr("Toyota Security Key Manager"), tr("RUN"));
+  connect(tskBtn, &ButtonControl::clicked, [this, tskBtn]() {
+    if (ConfirmationDialog::confirm(tr("Run Toyota Security Key Manager and reboot?"), tr("RUN"), this)) {
```

Comment:
also as discussed on discord, we must make sure the button is not enabled when offroad, but also make sure that even after the confirmation dialogue is closed, it will NOT run unless its still offroad